site/content/en/docs/tasks/run/multikueue/_index.md

```diff
@@ -1,7 +1,27 @@
 ---
-title: "Run Workloads in MultiKueue"
-weight: 2
+title: "Using MultiKueue"
+linkTitle: "MultiKueue"
+weight: 8
 date: 2024-09-25
 description: >
   The workloads that support MultiKueue.
 ---
+
+This page explains how to run tasks in MultiKueue environment.
+
+For more details about the MultiKueue check the concepts section for a [MultiKueue overview](/docs/concepts/multikueue/). 
+
+
+## Before you begin
+
+Check the [MultiKueue installation guide](/docs/tasks/manage/setup_multikueue) on how to properly setup MultiKueue clusters.
```

Comment:
Please also add that for the integration we recommend using at least Kueue 0.8.1, and JobSet 0.5.2.
Refer to the docs for installing a JobSet.